ydb/docs/en/core/yql/reference/syntax/delete.md

```diff
@@ -37,3 +37,18 @@ $to_delete = (
 DELETE FROM my_table ON
 SELECT * FROM $to_delete;
 ```
+
+## BATCH DELETE {#batch-delete}
```

Comment:
1. Make all these separate files (i.e.,`batch-delete.md`)
2. Add cross-links between normal `delete.md` and `batch-delete.md` in a `## See also` section at the end of the file
3. Make the first mention of `DELETE` in `batch-delete.md` a link to `delete.md`.
4. Repeat all of this for updates.